src/main/java/net/sf/jabref/event/ChangeFieldEvent.java

```diff
@@ -0,0 +1,22 @@
+package net.sf.jabref.event;
+
+
+public class ChangeFieldEvent {
+
+    private final String fieldName;
+    private final String value;
+
+    public ChangeFieldEvent(String fieldName, String value) {
```

Comment:
Could you rename `value` to `newValue`? Also at the getter? Otherwise, it is not straight forward understandable whether it is the new value or the old value.

(One could ask for adding `oldValue` and `bibEntry` as paramter. Since this is not yet required, we can add it )later.